DirectProgramming/DPC++/ProjectTemplates/cmake-gpu/License.txt

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2020, Intel Corporation. All rights reserved.
```

Comment:
Please use this license
https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt